pkgs/games/fteqw/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, stdenv
+, pkgs
```

Comment:
```suggestion
```
Please list all inputs individually.

Reply:
Is this ideal even with inputs that differ between packages? I can of course still do it, but the header list will be very long.

Reply:
I've changed this now.